cmd/kubeadm/app/master/manifests.go

```diff
@@ -296,6 +297,24 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		"--allow-privileged",
 	)
 
+	if cfg.AuthorizationMode != "" {
+		switch cfg.AuthorizationMode {
+		case authorizer.ModeABAC:
+			command = append(command, "--authorization-mode="+cfg.AuthorizationMode,
+				"--authorization-policy-file="+kubeadmapi.GlobalEnvParams.KubernetesDir+"/policy_file.json",
+			)
+		case authorizer.ModeRBAC:
+			command = append(command, "--authorization-mode="+cfg.AuthorizationMode,
+				"--runtime-config=rbac.authorization.k8s.io/v1alpha1",
```

Comment:
PR is open to add the beta version. https://github.com/kubernetes/kubernetes/pull/39625